src/ui/public/vis_maps/kibana_map.js

```diff
@@ -501,7 +509,10 @@ class KibanaMap extends EventEmitter {
       if (!centerFromUIState || centerFromMap.lon !== centerFromUIState[1] || centerFromMap.lat !== centerFromUIState[0]) {
         uiState.set('mapCenter', [centerFromMap.lat, centerFromMap.lon]);
       }
-    });
+    }
+
+    this.on('dragend', persistMapStateInUiState);
+    this.on('zoomend', persistMapStateInUiState);
```

Comment:
dragend and zoomend are triggered by human UI input. moveend was also triggered by auto-changes in size (e.g. caused by panels on the page sliding open and closed). This caused small jitters of the map-center, causing the appStatus to turn to dirty after the user opened the Reporting panel or the Save panel.